cmd/svcat/binding/get_cmd_test.go

```diff
@@ -0,0 +1,165 @@
+			wantError:    false,
+		},
+		{
+			name:         "get all existing bindings with json output format",
+			fakeBindings: []string{"myfirstbinding", "mysecondbinding"},
+			bindingName:  "",
+			outputFormat: "json",
+			expected:     "{\n   \"metadata\": {},\n   \"items\": [\n      {\n         \"metadata\": {\n            \"name\": \"myfirstbinding\",\n            \"namespace\": \"" + namespace + "\",\n            \"creationTimestamp\": null\n         },\n         \"spec\": {\n            \"instanceRef\": {},\n            \"externalID\": \"\"\n         },\n         \"status\": {\n            \"conditions\": null,\n            \"asyncOpInProgress\": false,\n            \"reconciledGeneration\": 0,\n            \"orphanMitigationInProgress\": false,\n            \"unbindStatus\": \"\"\n         }\n      },\n      {\n         \"metadata\": {\n            \"name\": \"mysecondbinding\",\n            \"namespace\": \"" + namespace + "\",\n            \"creationTimestamp\": null\n         },\n         \"spec\": {\n            \"instanceRef\": {},\n            \"externalID\": \"\"\n         },\n         \"status\": {\n            \"conditions\": null,\n            \"asyncOpInProgress\": false,\n            \"reconciledGeneration\": 0,\n            \"orphanMitigationInProgress\": false,\n            \"unbindStatus\": \"\"\n         }\n      }\n   ]\n}",
+			wantError:    false,
+		},
+		{
+			name:         "get all existing bindings with yaml output format",
+			fakeBindings: []string{"myfirstbinding", "mysecondbinding"},
+			bindingName:  "",
+			outputFormat: "yaml",
+			expected:     "items:\n- metadata:\n    creationTimestamp: null\n    name: myfirstbinding\n    namespace: " + namespace + "\n  spec:\n    externalID: \"\"\n    instanceRef: {}\n  status:\n    asyncOpInProgress: false\n    conditions: null\n    orphanMitigationInProgress: false\n    reconciledGeneration: 0\n    unbindStatus: \"\"\n- metadata:\n    creationTimestamp: null\n    name: mysecondbinding\n    namespace: " + namespace + "\n  spec:\n    externalID: \"\"\n    instanceRef: {}\n  status:\n    asyncOpInProgress: false\n    conditions: null\n    orphanMitigationInProgress: false\n    reconciledGeneration: 0\n    unbindStatus: \"\"\nmetadata: {}\n",
+			wantError:    false,
+		},
+		{
+			name:         "get all existing bindings with table output format",
+			fakeBindings: []string{"myfirstbinding", "mysecondbinding"},
+			bindingName:  "",
+			outputFormat: "table",
+			expected:     "       NAME         NAMESPACE   INSTANCE   STATUS  \n+-----------------+-----------+----------+--------+\n  myfirstbinding    " + namespace + "                        \n  mysecondbinding   " + namespace + "                        \n",
+			wantError:    false,
+		},
+		{
+			name:         "get existing binding with unknown output format",
+			fakeBindings: []string{"mybinding"},
+			bindingName:  "mybinding",
+			outputFormat: "unknown",
+			expected:     "",
+			wantError:    false,
```

Comment:
haha, I would have expected this to be `wantError: true`! You may have found a bug. :-) We can follow up on that after this PR is merged. 💯

Reply:
Do you think it is necessary to open an issue to track this?

Reply:
Yeah a bug is a good idea. Feel free to make one yourself and reference this test, or I will do that later today.